src/test/suite/fixtures/recorded/actions/bringAirAndBatAndCap.yml

```diff
@@ -1,8 +1,9 @@
-spokenForm: bring air and bat and cap
```

Comment:
New recorded test syntax just stores an exact snapshot of the command argument that comes in to the cursorless command

Reply:
Sounds good. Really inline with our discussion about that the tests should only capture what the user actually said.